src/Http/Controllers/ProfileInformationController.php

```diff
@@ -14,15 +14,14 @@ class ProfileInformationController extends Controller
      *
      * @param  \Illuminate\Http\Request  $request
      * @param  \Laravel\Fortify\Contracts\UpdatesUserProfileInformation  $updater
-     * @return \Illuminate\Http\Response
+     * @return \Laravel\Fortify\Contracts\ProfileInformationUpdateResponse
      */
-    public function update(Request $request,
-                           UpdatesUserProfileInformation $updater)
-    {
+    public function update(
+        Request $request,
+        UpdatesUserProfileInformation $updater
+    ) {
```

Comment:
Please undo the styling changes here.

Reply:
fixed.

Reply:
It's still incorrect. Please revert it to the original code.